### PR TITLE
.NET Core 3.1 Support

### PR DESCRIPTION
**Pull request by progdogusmc**
### Changes

Modified WinForms and WPF project files to build both .NET Framework 4.6.2 and .NET Core 3.1 DLLs.

Modified WinForms and WPF nuspec files to publish packages that support both .NET Framework 4.6.2 and .NET Core 3.1.

### References

https://github.com/auth0/auth0-oidc-client-net/issues/147

### Testing

No substantial changes were made to functionality. Existing WinForms and WPF test apps continue to operate as expected. Created a WPF.Core test app using the existing WPF test app files. .NET Core DLLs work as expected. WPF.Core test app is attached as a ZIP.

[WPF.Core.zip](https://github.com/auth0/auth0-oidc-client-net/files/4407196/WPF.Core.zip)

### Checklist

- [x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
- [x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
- [x] All existing and new tests complete without errors
- [x] All code guidelines in the [CONTRIBUTING documentation](../CONTRIBUTING.md) have been run/followed

### CODE REVIEW

File: nuget/Auth0.OidcClient.WPF.nuspec [hunk 1]
```diff
@@ -12,12 +12,15 @@
...
     <releaseNotes>
+      Version 3.1.4
+      - Now supports .NET Core 3.1 projects
+
       Version 3.1.3
       - WPF version is now actually strong-named.
 
       Version 3.1.2
       - Allow ID tokens "issued at" (iat) claims from "the future" to allow
-        slow local clocks on mobile and desktop devices.
```
Comment (damieng) [inline nuget/Auth0.OidcClient.WPF.nuspec]: Can you revert these changes. I know they shouldn't be space sensitive which is why VS reformats them but they actually are in NuGet.

Comment (progdogusmc) [reply nuget/Auth0.OidcClient.WPF.nuspec]: I will close this pull request and submit a new one without the whitespace changes
